pkg/ebpf/c/v4events.bpf.c

```diff
@@ -32,10 +32,11 @@ struct conntrack_key {
    __u32 dest_ip;
    __u16 dest_port;
    __u8  protocol;
+   __u32 owned_ip;
```

Comment:
Let's make this `owner_ip` as well. We're also missing the corresponding change in `v6events.bpf.c`